action.yml

```diff
@@ -108,14 +148,32 @@ runs:
             "Refer to the documentation for migration guidance: https://github.com/camunda-community-hub/community/blob/main/maintainers-reviewers/RELEASE.MD#migration-to-sonatype-central-portal--how-to"
         fi
 
+        # Fallback to the deprecated profile if namespace is not migrated yet
+        # This is a temporary measure to smooth migration by letting users switch to the @v2 version of the action and new profile immediately
+        # The fallback will be removed namespace by namespace as migration completes
+        group_id=$(mvn help:evaluate -Dexpression=project.groupId -q -DforceStdout)
```

Comment:
In https://github.com/camunda-community-hub/community-action-maven-release/pull/73/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R99 I had to sanitize the output of Maven from invisible control characters to allow successful Bash `if` condition matching. Did you test whether this is needed here as well?

Reply:
Yes, in the provided [tests](https://github.com/camunda/team-infrastructure/issues/833#issuecomment-2936908285), and locally also. I never had the issue, but my code check the prefix only.

Reply:
Makes sense, thx.